cmd/mattermost/commands/extract_content.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/shared/mlog"
+)
+
+var ExtractContentCmd = &cobra.Command{
+	Use:     "extract-documents-content",
+	Short:   "Extracts the documents content",
+	Long:    "Extracts the documents content and stores it in the database for document searche",
+	Example: "extract-documents-content --from=12345",
+	RunE:    extractContentCmdF,
+}
+
+func init() {
+	ExtractContentCmd.Flags().Int64("from", 0, "The timestamp of the earliest file to extract, expressed in seconds since the unix epoch.")
+	ExtractContentCmd.Flags().Int64("to", model.GetMillis(), "The timestamp of the latest file to extract, expressed in seconds since the unix epoch.")
+	RootCmd.AddCommand(ExtractContentCmd)
+}
+
+func extractContentCmdF(command *cobra.Command, args []string) error {
+	a, err := InitDBCommandContextCobra(command)
+	if err != nil {
+		return err
+	}
+	defer a.Srv().Shutdown()
+
+	if !*a.Config().FileSettings.ExtractContent || !a.Config().FeatureFlags.FilesSearch {
+		return errors.New("ERROR: Document extraction is not enabled")
+	}
+
+	startTime, err := command.Flags().GetInt64("from")
+	if err != nil {
+		return errors.New("from flag error")
+	}
+	if startTime < 0 {
+		return errors.New("from must be a positive integer")
+	}
+
+	endTime, err := command.Flags().GetInt64("to")
+	if err != nil {
+		return errors.New("to flag error")
```

Comment:
small nit, maybe quote the word 'to' so that the message is clearer?

Reply:
Done!